packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -297,4 +301,76 @@ class WKWebView {
   Future<void> loadRequest(NSUrlRequest request) {
     throw UnimplementedError();
   }
+
+  /// Loads the contents of the specified HTML string and navigates to it.
+  Future<void> loadHtmlString(String string, String? baseUrl) {
+    throw UnimplementedError();
+  }
+
+  /// Loads the web content from the specified file and navigates to it.
+  Future<void> loadFileUrl(String url, String readAccessUrl) {
```

Comment:
This method actually takes `NSURL`s and I considered implementing them. However, they would just be serialized into Strings, so I didn't think it was a necessary extra step.

Reply:
Makes sense. We will need to pay attention to error handling though when implementing the native side (which we haven't always been good about in our plugins) since String->NSURL conversion will fail if the URL isn't valid per the RFC Apple used. (Which isn't the same as `Url` unfortunately, so we shouldn't use that here unfortunately.)